.idea/src/Lesson18/Task2/Map.java

```diff
@@ -0,0 +1,28 @@
+package Lesson18.Task2;
+
+import java.time.DayOfWeek;
+import java.util.HashMap;
+
+public class Map<K, V> {
```

Comment:
the map must be like the java.util.Map. So you can use it for saving pairs key - value.